app/assets/stylesheets/frontend/reset/_reset.scss

```diff
@@ -30,3 +30,10 @@ table, caption, tbody, tfoot, thead, tr, th, td {
   font-weight: normal;
 }
 
+strong {
+  font-weight: bold;
+}
```

Comment:
Should have linked that commit to [static’s reset file](https://github.com/alphagov/static/blob/master/app/assets/stylesheets/helpers/_reset.scss), where you [purposefully added this in the past](https://github.com/alphagov/static/commit/2184c82e45c186c71f12b026209eca610f816228).